### DISCUSSION TITLE
The kernel crash :(

### DISCUSSION THREAD

**Discussion by MinnTrit** [mrdbourke/pytorch-deep-learning#732] [General]
I'm currently working in vs code using with the 2nd chapter in the video, my code works nice but I can't visualize it, every time I make a visualization, I had the following error: 

"The Kernel crashed while executing code in the the current cell or a previous cell. Please review the code in the cell(s) to identify a possible cause of the failure. Click [here](https://aka.ms/vscodeJupyterKernelCrash) for more info. View Jupyter [log](command:jupyter.viewOutput) for further details.", 
![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/151976884/714b1e2a-ddb8-46f1-8a19-b133a79bf338)

Does anybody know how to fix it, I couldn't find the solution on ChatGPT as well :<

**Comment by shisirkha** [comment]
The problem you are facing is not simple but it's your python environment kernel problem. i think you need to give me the jupyter notebook logs for me to find a fix

**Reply by shisirkha** [reply]
Also use google colab if it happen everytime I think your pc ram is not enough to run this simple program.

**Comment by Abosamir** [comment]
i got the same error and i don't know yet how to solve it.
can someone help?

**Reply by Abosamir** [reply]
this problem was solved after updating NumPy and matplotlib versions

**Comment by LuluW8071** [comment]
Maybe lower your python version to 3.10
I had same type of random error on 3.11
